Extensions/TextObject/textruntimeobject-pixi-renderer.js

```diff
@@ -76,3 +76,49 @@ gdjs.TextRuntimeObjectPixiRenderer.prototype.getWidth = function() {
 gdjs.TextRuntimeObjectPixiRenderer.prototype.getHeight = function() {
     return this._text.height;
 };
+
+/**
+ * Get scale of the text.
+ */
+gdjs.TextRuntimeObjectPixiRenderer.prototype.getScale = function() {
```

Comment:
This is not used right? Can we remove this to decrease the API surface and avoid potential confusions? :)

Reply:
done